app/models/item.rb

```diff
@@ -18,6 +22,14 @@ class Item < ActiveRecord::Base
   enum edit_reasons: [:donation, :purchase, :correction, :order_adjustment]
   enum edit_methods: [:add, :subtract, :new_total]
 
+  def self.find_any(id)
+    unscoped.find(id)
+  end
+
+  def self.find_deleted(id)
+    find_any(id)
+  end
+
```

Comment:
find_deleted and find_any appear to be the same. I would think find_deleted would scope to only soft deleted items.